ci/windows_ci_steps.ps1

```diff
@@ -0,0 +1,16 @@
+#powershell.exe -Command
```

Comment:
For the maintainers to consider: we added this version of the `window_ci_steps` script in case we wanted to converge on using powershell only instead of a mix of bash and powershell, up to y'all, we can remove it if we want to stick with bash scripts

Reply:
If powershell is desired, we can refactor with an "invoke bazel" function etc. so we dont have to keep rewriting the error code checking

Reply:
I'm OK either way as long as we have one canonical solution.

Reply:
ok, removing this one